scripts/migrate_quotes.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import csv
+import os
+import pathlib
+
+import requests
+
+NB2_PORT = os.environ.get("NB2_PORT", "8000")
+
+
+def get_url(url):
+    return f"{'http' if parsed_args.use_http else 'https'}://127.0.0.1:{NB2_PORT}{url}"
+
+
+def post(url, data):
+    response = requests.post(get_url(url), data)
+    if response.status_code > 400 and response.status_code != 409:
+        raise Exception(response.json())
+    return response
+
+
+if __name__ == "__main__":
+    # parser
+    parser = argparse.ArgumentParser(description="Beep borp I migrate data for the cutest bot ever")
+    parser.add_argument(
+        "source_file",
+        type=open,
+        help="A csv where each line denotes a user's slack_id, ghost_id, display_name, first_name and file(s) associated with them.",  # noqa: E501
+    )
+    parser.add_argument(
+        "quotes_folder", help="The source folder of where the quote files are", type=pathlib.Path
+    )
+    parser.add_argument("--use-http", help="Use http instead of https", action="store_true")
+
+    parsed_args = parser.parse_args()
+    source_file = parsed_args.source_file
+
+    csvreader = csv.DictReader(source_file, delimiter=",")
+    for row in csvreader:
+        quote_filename = row.pop("file")
+        post("/people", row)
```

Comment:
We don't have to worry about empty columns because the api takes care of that right?